deployment/src/main/java/org/wso2/carbon/testgrid/deployment/deployers/PuppetDeployer.java

```diff
@@ -74,14 +83,23 @@ public Deployment deploy(Deployment deployment) throws TestGridDeployerException
      * @param location location of k8s.properties file
      * @return String value of KUBERNETES_MASTER property
      */
-    private String getKubernetesMaster(String location){
+    private String getKubernetesMaster(String location) {
         Properties prop = new Properties();
+        InputStream inputStream = null;
         try {
-            InputStream inputStream = new FileInputStream(location);
+            inputStream = new FileInputStream(location);
```

Comment:
We can use Java 8 try with resource [1].

[1] https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Fixed with 2a2b4960dc5c7102670110a20c401277f24af375